design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Azure Simple
+draft: true
+date: 2019-05-18T13:09:18.252Z
+author:
+  name: Rachel Weil
+  github-username: hxlnt
+sourceUrl: https://github.com/microsoft/cobalt/
+tags:
+  - app-insights
+  - app-gateway
+  - azure
+  - monitoring
+  - traffic-manager
+targetUser:
```

Comment:
How do you want to indicate which sections are auto generated?

Reply:
**THIS CONTENT IS NOT AUTOGENERATED.** 
Latest commit adds this tag around content that is not autogenerated.
**//**